documentation/concepts/!concept-template.md

```diff
@@ -0,0 +1,103 @@
+---
+title: {Concept Name}
+---
+<!-- template preamble -->
+
+## Template preamble (remove!)
+
+A concept page explains some aspect of the API Management Platform. They can be
+thought of as extended definitions of major abstractions, such as a resource
+(like Products) or higher-level concept (like an API Gateway) or an overview of
+the Platform. Concept pages provide essential background, enabling readers to
+understand the Platform, and thereby navigate tasks to achieve their goals.
+Typically, concept pages don't include sequences of steps, but instead provide
+links to tasks or tutorials.
+
+Considerations for page audience:
+
+- Layer depths of explanation: Start with simple, high-level explanations for
+  non-technical readers, then progress into detailed technical explanations.
+- If that isn't enough, then consider dividing the concept page into sections
+  for different audiences.
+
+To write a new concept page, copy this file. All text in {curly brackets} should
+be replaced or removed.
+
+For more information on concept pages, visit the [Good Docs Project](https://gitlab.com/tgdp/templates/-/tree/main/concept).
+
+---
+
+<!-- overview -->
+
+{A summary paragraph introducing a concept, explaining its importance or
+relevance, and providing an overview of the content that will be covered
+in the document (scope).}
+
+{Typical wording to use is:}
+
+This article explains the basics of {concept} and how it works in {the tool or context}.
+
+{Then include a paragraph with a definition of the concept you are explaining.
+If more definitions are needed, include those definitions here as a bulleted list.}
```

Comment:
or can link out to glossary?

Reply:
Template may also benefit from a place to link out externally to more information...E.g. for Kong Plugins, maybe we link out to Kong Docs?

Reply:
> Template may also benefit from a place to link out externally to more information...E.g. for Kong Plugins, maybe we link out to Kong Docs?

nvm - I see you have a spot for this below!

Reply:
@mlevando - The glossary will have short, concise definitions. If a concept page and glossary entry both exist for a term, the concept page is going to be more in depth. So at this point, I don't think we should ever be linking to the glossary (tooltips will make glossary definitions available in situ). 

Maybe this could change if we decided to have longer definitions. But I'd say better to create a brief concept page.

Reply:
i was referring specifically to the 'if more definitions are required' - would any of these definitions ever be in glossary instead? @rustyjux

Reply:
For sure. The idea of the concept page is to give the good fullsome explanation of the core concept *on this page*. This might require more than a single definition. Introducing and explaining the concept will often involve repeating the concise definition from the glossary for the core (or closely relately) terms covered by the concept. 

For adjacent / less-closely-related concepts or terms, the appropriate move would be to include the {{ glossary_tooltip }} reference, which provides the glossary definition and links to the best reference page (if one exists). I need to add example use of {{ glossary_tooltip }} to all the templates, as this did integration did not exist when I drafted the templates.